app/templater/lib/templater/__init__.py

```diff
@@ -20,23 +20,30 @@ def __init__(self):
 
 class TemplateRenderer(object):
 
+    MULTILINE_PREFIX = "&^<\\multiline>"
```

Comment:
Please, add examples of templates and tables for this feature and update README.md with its description